src/Middleware/Diagnostics.Abstractions/src/IDeveloperPageExceptionFilter.cs

```diff
@@ -9,7 +9,7 @@
 namespace Microsoft.AspNetCore.Diagnostics;
 
 /// <summary>
-/// Provides an extensiblity point for changing the behavior of the DeveloperExceptionPageMiddleware.
+/// Provides an extensibility point for changing the behavior of the <see cref="DeveloperExceptionPageMiddleware"/>.
```

Comment:
```suggestion
/// Provides an extensibility point for changing the behavior of the DeveloperExceptionPageMiddleware.
```

The issue is that IDeveloperPageExceptionFilter is defined in Microsoft.AspNetCore.Diagnostics.Abstractions instead of Microsoft.AspNetCore.Diagnostics and the reference points the other way. Does anyone know a way to link to unreferenced types?

Reply:
Just use `href`.

Reply:
For `href` I think you'd need to link to a valid fully-qualified URI for it to be useful.

Reply:
It would be a bug in the doc engine.

Reply:
Tools other than those that power `docs.microsoft.com` also process XML documentation comments, so the markup needs to point to something concrete. `href="DeveloperExceptionPageMiddleware"` by itself doesn't point to anything specific as it's not a fully-qualified URL (like `https://docs.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.diagnostics.developerexceptionpagemiddleware).

While the "doc engine" for `docs.microsoft.com` could be smart enough to special-case it, that would only fix that website, and wouldn't be useful for other tooling (such as Visual Studio Intellisense, Visual Studio Code, Rider, etc.) that might render the link to the user in some other context, as it wouldn't go anywhere.

Reply:
You would make the same argument to shoot down the concept of a relative URL wholesale.  Also, an improvement to one site is still an improvement.

Reply:
An improvement to one site at the expense of a dead-end link that does nothing in `N > 1` other tools isn't a net-positive improvement though.

Reply:
Why would the paths to these 2 documents be different in any tool?  They are logically in the same place.

Reply:
`DeveloperExceptionPageMiddleware` doesn't point anywhere though - if you put that into your URL bar in your browser, you'd get a 404 or the results of a search engine query for that literal string. If you did an HTTP GET request to it with `HttpClient`, you wouldn't get anything.

It's a relative URI, not an absolute URI, and without tooling special-casing to go and find what's _meant_, it won't find anything useful.

Reply:
I think the crux of the issue is what is the URI relative _to_? The file? The namespace? The assembly?

Seems to me that it's contextual and ambiguous (where a `cref` isn't), which is why the `href` doesn't seem correct IMO.

If there's something about how that's treated universally that I'm not aware of that makes it "just work", then fair enough.

Reply:
The URI is relative to the base path of the referrer.

Reply:
But what is the referrer? Seems to be something that would be contextual to me.

Reply:
The referrer  depends on the user agent.  But since both documents are in the same context, this question is irrelevant.

Reply:
Not really - what if the User Agent is Visual Studio's Intellisense pop-up - what would it open if you clicked on the link?

The types are in different assemblies, they just happen to be in the same namespace. I don't see how this would work in a context like this where I click `IDictionary<TKey, TValue>` and it opens the assembly reference for the `IDictionary<TKey, TValue>` type in the `System.Runtime` assembly. How is the user agent going to know where the type lives _just_ from the information in the link?

![image](https://user-images.githubusercontent.com/1439341/144062891-f7148814-b213-4994-9b53-6f16b405b520.png)

Reply:
I was corrected to use the full type rather than the bare type name.  Intellisense should just find the type in the assemblies referred to by the project.  If it is unable to do that for any reason, it should disable relative `href` links, rendering just the text (which is what you would like to happen everywhere).  That would mean the link would be OK in the browser and harmless for Intellisense.

Reply:
I'm going to just apply my suggestion for now and take the spelling fix. Sorry @yecril71pl. While I agree it would be nice if all the tooling worked well with unreferenced types, I do see why it's a difficult problem to solve, and I don't think we can solve it here.